### ISSUE TITLE
Custom open browser command 

### ISSUE THREAD

**Issue by monde** [okta/okta-aws-cli#134]
We are using https://github.com/pkg/browser to pop open the web browser when the flag `--open-browser` is set. It does as simple an open URL command as possible from a shell for many OS/browser combinations. For instance, on OSX it is just calling open, see #131 . Golang has exec process capability https://pkg.go.dev/os/exec , we should allow to the user to set the exec command to open the browser in their environment. Then they could customize opening the browser on their OS and browser profile.

Incognito exaple
```
$ okta-aws-cli \
  --open-browser \
  --open-browser-command "open -na 'Google Chrome' --args -incognito"
```

Named profile example, exec'ing chrome directly on OSX
```
$ okta-aws-cli \
  --open-browser \
  --open-browser-command "/Applications/Google\ Chrome.app/Contents/MacOS/Google\ Chrome --args --profile-directory='Profile 1'"
```

**Comment by monde** [comment]
Okta internal reference https://oktainc.atlassian.net/browse/OKTA-653005

**Comment by monde** [comment]
Released in v2.0.0 https://github.com/okta/okta-aws-cli/releases/tag/v2.0.0
